eng/clang-wrapper.sh

```diff
@@ -22,4 +22,4 @@ case $CLANG_NAME in
         ;;
 esac
 
-$CLANG_CC $EXTRA_ARGS $@
+$CLANG_CC $EXTRA_ARGS "$@"
```

Comment:
should we wrap `CLANG_CC` as well, in case it has spaces?

```suggestion
"$CLANG_CC" $EXTRA_ARGS "$@"
```